# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../nginx/default_ssl_ciphers.patch
# Copyright (C) 2012 - 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

http://matt.io/entry/ur

--- ./conf/nginx.conf.orig	2012-04-25 11:40:14.000000000 +0200
+++ ./conf/nginx.conf	2012-04-25 11:47:11.000000000 +0200
@@ -105,7 +105,7 @@
     #    ssl_session_cache    shared:SSL:1m;
     #    ssl_session_timeout  5m;
 
-    #    ssl_ciphers  HIGH:!aNULL:!MD5;
+    #    ssl_ciphers  HIGH:!aNULL:!MD5:!kEDH;
     #    ssl_prefer_server_ciphers  on;
 
     #    location / {

